### ISSUE TITLE
Support for local prediction

### ISSUE THREAD

**Issue by michelole** [microsoft/dstoolkit-mlops-base#66]
Currently, [score.py](https://github.com/microsoft/dstoolkit-mlops-base/blob/main/src/score.py) is the only "src" file that has no main method and thus cannot be easily run locally.

It would be nice to to have that to ease testing during development.

**Comment by FlorianPydde** [comment]
Indeed, the debugging is not the most user-friendly approach. The fastest would be to test it with a local model or use this: https://docs.microsoft.com/en-us/azure/machine-learning/how-to-troubleshoot-deployment-local
